spec/suites/map/handler/Map.DragSpec.js

```diff
@@ -132,7 +132,7 @@ describe("Map.Drag", function () {
 			});
 
 			var originalCenter = L.latLng(0, 0);
-			map.setView(originalCenter, 1);
+			map.setView(originalCenter.clone(), 1);
```

Comment:
Maybe `clone()` should be not necessary here. It should be cloned in the backend logic of `setView`.

Reply:
It's necessary because it makes sure the tests below compare with the original value regardless of Leaflet's internal implementation it is testing against.

Reply:
Ah yes I understand